packages/itwinui-css/backstop/tests/progress-indicator.html

```diff
@@ -1099,5 +1104,13 @@ <h2>Overlay</h2>
         </div>
       </div>
     </section>
+
+    <script type="module">
+      (async () => {
+        if (!HTMLElement.prototype.hasOwnProperty('inert')) {
```

Comment:
What does this condition do? Why is it required?

Also, what is `HTMLElement.prototype` supposed to return? Because VSCode was not autocompleting `hasOwnProperty`

![image](https://user-images.githubusercontent.com/45748283/205101700-3703cc0c-aa9f-40a9-b368-fdfda60f97a7.png)